apidoc/Titanium/UI/ActivityIndicator.yml

```diff
@@ -134,7 +134,7 @@ properties:
     since: "2.1.0"
     type: String
     default: <pre>#fff</pre>
-    platforms: [mobileweb]
+    platforms: [mobileweb,iOS]
```

Comment:
@hansemannn @cheekiatng updated and sent https://github.com/appcelerator/titanium_mobile/pull/7041